Dockerfile-alpine.template

```diff
@@ -2,6 +2,9 @@ FROM alpine:3.4
 
 ENV NPM_CONFIG_LOGLEVEL info
 ENV NODE_VERSION 0.0.0
+ENV ARCH x64
```

Comment:
Using an explicit `ENV` in this way will not be acceptable for the official images.

I would recommend using the output of `dpkg --print-architecture` or `apk --print-arch` in order to determine the architecture of the "current" platform.  For example, see https://github.com/docker-library/golang/blob/cc59d934fca2e3a7dd8ebcd0eee82f34589f3bd3/1.9-rc/Dockerfile#L16-L27.